airbyte-webapp/src/components/ui/DatePicker/index.ts

```diff
@@ -1 +1,3 @@
-export { DatePicker } from "./DatePicker";
+import React from "react";
+
+export default React.lazy(() => import("./DatePicker"));
```

Comment:
Could we just export this normally here, and defer the lazy import to the consumer?